src/app/productConfig/productConfigModal.tpl.html

```diff
@@ -35,20 +35,21 @@ <h4 class="panel-title border-bottom-small" translate>
             <!--</div>-->
 
             <div data-toggle="buttons" ng-class="{'has-error': ($ctrl.itemConfigForm.amount | showErrors)}">
-              <label class="btn btn-radio" ng-click="$ctrl.itemConfig.amount = 50" ng-class="{'active': $ctrl.itemConfig.amount === 50}"><span class="number">$50</span></label>
-              <label class="btn btn-radio" ng-click="$ctrl.itemConfig.amount = 100" ng-class="{'active': $ctrl.itemConfig.amount === 100}"><span class="number">$100</span></label>
-              <label class="btn btn-radio" ng-click="$ctrl.itemConfig.amount = 250" ng-class="{'active': $ctrl.itemConfig.amount === 250}"><span class="number">$250</span></label>
-              <label class="btn btn-radio" ng-click="$ctrl.itemConfig.amount = 500" ng-class="{'active': $ctrl.itemConfig.amount === 500}"><span class="number">$500</span></label>
-              <label class="btn btn-radio" ng-click="$ctrl.itemConfig.amount = 1000" ng-class="{'active': $ctrl.itemConfig.amount === 1000}"><span class="number">$1000</span></label>
-              <label class="btn btn-radio" ng-click="$ctrl.itemConfig.amount = 5000" ng-class="{'active': $ctrl.itemConfig.amount === 5000}"><span class="number">$5000</span></label>
+              <label class="btn btn-radio" ng-click="$ctrl.changeAmount(a)" ng-class="{'active': $ctrl.itemConfig.amount === a}" ng-repeat-start="a in $ctrl.selectableAmounts">
+                <span class="number">${{a}}</span>
+              </label>
+              <span ng-repeat-end></span>
```

Comment:
idk.  What they have is valid html.  ng-repeat just removes the spacing between each element.  I think this is the best way to resolve that.